### ISSUE TITLE
New onboarding: show "are you sure you wanna leave?" prompt

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#50119]
When closing `/new` onboarding flow in browser, we could show the standard "are you sure?" browser prompt:

<img width="601" alt="Screenshot 2021-02-16 at 11 51 34" src="https://user-images.githubusercontent.com/87168/108046570-6db05f00-704d-11eb-995b-86e4cf1e7bcd.png">

Not sure if we can affect the copy here as it might come from the browser?

Example from Gutenberg:

https://github.com/WordPress/gutenberg/blob/004a32ec09d5b789be694d85725fe49cca385f08/packages/editor/src/components/unsaved-changes-warning/index.js

**Comment by simison** [comment]
@jasmussen can you catch up with @nuriapenya on this one? Thx!

**Comment by nuriapenya** [comment]
Heya 👋🏼 I think we are looking into two different things here.

@simison, what you are suggesting is a way to prevent users from closing the onboarding at any step, right? This would be a new feature, since we don't have anything like this at the moment.

What @jasmussen is describing is a bug. Users should be able to go back to previous step while using the browser back button. I'd suggest to open a new issue for that one in particular.

**Comment by jasmussen** [comment]
Thanks for confirming Núria! I've created https://github.com/Automattic/wp-calypso/issues/98797 instead.

**Comment by simison** [comment]
@nuriapenya I don't really remember — this issue is from times working on `/new` flow circa 2021 and the flow hasn't existed for a long time, so I don't really remember. :-)

I think it's safe to close, feel free to re-open if you'd find it useful.

**Comment by nuriapenya** [comment]
> When closing /new onboarding flow in browser, we could show the standard "are you sure?" browser prompt:

You mean something like we do in the Editor, right?

![Image](https://github.com/user-attachments/assets/d9bf1bc0-7c89-4331-a2c3-cceb3212fd74)

I am not sure about it. Isn't it a bit intrusive? It will be impossible if the user wants to close the whole browser because that tab/popup will prevent it. Also, the user didn't "modify/add/edit" anything, really. Just selected some options. 

What are we trying to solve exactly?

If our goal is to make sure users don't lose track of things they already did during onboarding, I find more interesting an approach like GoDaddy where they ask you to pick up wherever you left off:

![Image](https://github.com/user-attachments/assets/e6f9c380-eebb-4694-8339-9974f8a86dd3)

**Comment by nuriapenya** [comment]
Omg @simison I didn't see the date 😂 I thought it was a new proposal -- eitherway, my answer stands for posterity haha

**Comment by simison** [comment]
Yeah, what Godaddy does with "pick up where you left?" is quite brilliant and I've suggested it couple times in past for our flows to solve "returning user" problems. I think UX like that would layer nicely with some ideas @alshakero described in P2 pdDR7T-242-p2

**Comment by nuriapenya** [comment]
As a follow up I created this issue, just to have this idea on the radar: https://github.com/Automattic/wp-calypso/issues/98798 I also think it has potential. Maybe we should check %%% people who close/drop-off during onboarding, and what happens if/when they come back.
